conan/tools/microsoft/msbuilddeps.py

```diff
@@ -164,7 +183,8 @@ def add_valid_ext(libname):
             'definitions': "".join("%s;" % d for d in cpp_info.defines),
             'compiler_flags': " ".join(cpp_info.cxxflags + cpp_info.cflags),
             'linker_flags': " ".join(cpp_info.sharedlinkflags),
-            'exe_flags': " ".join(cpp_info.exelinkflags)
+            'exe_flags': " ".join(cpp_info.exelinkflags),
+            'ca_exclude_dirs': '$(Conan%sIncludeDirectories)' % name if exclude_code_analisys else ''
```

Comment:
```suggestion
            'ca_exclude_dirs': '$(Conan{}IncludeDirectories)'.format(name) if exclude_code_analysis else ''
```

Typo in ``exclude_code_analysis``. Also the ``.format()`` helps dissambiguate precedence of % name vs if